bin/desi_compute_pixel_flatfield

```diff
@@ -458,20 +462,28 @@ n1=image.shape[1]
 b1=n1//2-pixel_band_width_for_initial_spectral_model//2
 e1=n1//2+pixel_band_width_for_initial_spectral_model//2
 band=image[:,b1:e1].copy()
-for loop in range(3) :
-    medspec=np.median(band,axis=1)
-    for i in range(band.shape[1]) :
-        band[:,i] *= np.median(band[:,i]/medspec)
-    medspec=np.median(band,axis=1)
+
+if np.any(np.isnan(band)) :
+    log.error("band image has nan values")
+    sys.exit(12)
+
+if 1 :
```

Comment:
Leftover debugging `if 1`?  i.e. let's either make this an option that we can genuinely turn on and off, or drop the `if 1` and indentation.